arch/x86/kernel/cpu/resctrl/rdtgroup.c

```diff
@@ -3473,9 +3481,16 @@ static int rdtgroup_mkdir_ctrl_mon(struct kernfs_node *parent_kn,
 	ret = 0;
 
 	rdtgrp->closid = closid;
+
+	ret = mkdir_rdt_prepare_rmid_alloc(rdtgrp);
+	if (ret)
+		goto out_id_free;
```

Comment:
at upstream the goto is out_closid_free

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev